src/main.rs

```diff
@@ -4,7 +4,7 @@ fn main() {
     let exit_code = match n2::run::run() {
         Ok(code) => code,
         Err(err) => {
-            println!("n2: error: {}", err);
+            eprintln!("n2: error: {}", err);
```

Comment:
I think this is undesirable, because we print other progress information to stdout.  Logging the same information to two different channels isn't a good idea.

Reply:
Ninja logs build progress/failures into stdout & usage errors (inccorect cli arguments, non-existent targets, etc.) into stderr.

Make logs build progress to stdout & build/usage errors to stderr.

("Make logs build progress", heh. What's a verb? What's a noun? What's an adjective?)

I'm not sure which of the two approaches I like more, but they both are reporting **usage** errors to stderr. I thought this code was for handling precisely that, but if its not the case and build failures also get propagated here, I can try and separate those, if needed.